src/functionalTest/java/uk/gov/hmcts/dts/fact/admin/list/AdminAreasOfLawEndpointTest.java

```diff
@@ -43,4 +60,170 @@ public void shouldBeForbiddenForGettingAllCourtAreasOfLaw() {
         );
         assertThat(response.statusCode()).isEqualTo(FORBIDDEN.value());
     }
+
+    @Test
+    public void shouldReturnAreaOfLaw() {
+        final Response response = doGetRequest(
+            ADMIN_AREAS_OF_LAW_ENDPOINT + AREA_OF_LAW_ID_HOUSING_POSSESSION,
+            Map.of(AUTHORIZATION, BEARER + authenticatedToken)
+        );
+        assertThat(response.statusCode()).isEqualTo(OK.value());
+        final AreaOfLaw areaOfLaw = response.as(AreaOfLaw.class);
+        assertThat(areaOfLaw).isNotNull();
+        assertThat(areaOfLaw.getId()).isEqualTo(AREA_OF_LAW_ID_HOUSING_POSSESSION);
+    }
+
+    @Test
+    public void shouldRequireATokenWhenGettingAreaOfLaw() {
+        final Response response = doGetRequest(ADMIN_AREAS_OF_LAW_ENDPOINT + AREA_OF_LAW_ID_HOUSING_POSSESSION);
+        assertThat(response.statusCode()).isEqualTo(UNAUTHORIZED.value());
+    }
+
+    @Test
+    public void shouldBeForbiddenForGettingAreaOfLaw() {
+        final Response response = doGetRequest(
+            ADMIN_AREAS_OF_LAW_ENDPOINT + AREA_OF_LAW_ID_HOUSING_POSSESSION,
+            Map.of(AUTHORIZATION, BEARER + forbiddenToken)
+        );
+        assertThat(response.statusCode()).isEqualTo(FORBIDDEN.value());
+    }
+
+    @Test
+    public void shouldReturnAreaOfLawNotFound() {
+        final Response response = doGetRequest(
+            ADMIN_AREAS_OF_LAW_ENDPOINT + "12345",
+            Map.of(AUTHORIZATION, BEARER + forbiddenToken)
+        );
+        assertThat(response.statusCode()).isEqualTo(FORBIDDEN.value());
```

Comment:
@Ayisha-Sharjeel  This test should be testing Not found but you are passing in a forbidden token and test it is forbidden. Forbidden is already tested in the previous test. You need to pass in a correct token with invalid ID in this test

Reply:
I've changed the test. thanks